docs/gallery.rst

```diff
@@ -128,12 +128,6 @@ Classification Visualizers
     :alt: Multiclass ROC Curves
     :target: api/classifier/rocauc.html#multi-class-rocauc-curves
 
-.. image:: api/target/images/class_balance_compare.png
```

Comment:
In the gallery documentation, I moved the Class Balance image from "Classification Visualizers" to under "Target Visualizers".